scripts/bash_completion/optirun

```diff
@@ -54,7 +54,8 @@ _optirun() {
           *)
```

Comment:
Shouldn't `--vgl-options` be handled here as well? (otherwise it will suggest other options